db/contacts.json

```diff
@@ -0,0 +1,94 @@
+{
+    "contacts": [{
+            "id": 0,
```

Comment:
Oops! Sorry we didn't catch this before. Your JSON should be an object (instead of an array) of objects with keys for each contact (just like it will be when we get it back from Firebase).